### PR TITLE
issue-5230

### PR DESCRIPTION
**Pull request by LionOnTheChase**
added traceid and spanid to decision log.

test for checking traceid in decision log is not passing

### CODE REVIEW

File: internal/internal.go [hunk 1]
```diff
@@ -519,6 +521,22 @@ func (p *envoyExtAuthzGrpcServer) log(ctx context.Context, input interface{}, re
 		info.Path = p.cfg.Path
 	}
 
+	sctx := trace.SpanFromContext(ctx).SpanContext()
+	if sctx.IsValid() {
+		info.TraceID = sctx.TraceID().String()
+		info.SpanID = sctx.SpanID().String()
+	}
+
+	p.manager.Logger().WithFields(map[string]interface{}{
```
Comment (ashutosh-narkar) [inline internal/internal.go:530]: We should remove this log as much of this is already logged at debug level in the `check` call. This is also resulting in the test failure.

File: internal/internal_test.go [hunk 2]
```diff
@@ -165,6 +167,38 @@ func TestCheckAllow(t *testing.T) {
 	}
 }
 
+func TestCheckTraceIDInAllowedRequest(t *testing.T) {
```
Comment (ashutosh-narkar) [inline internal/internal_test.go:170]: You could use something like [this](https://github.com/open-policy-agent/opa/blob/main/test/e2e/distributedtracing/distributedtracing_test.go#L148) as a reference to setup the unit test.

**Review by ashutosh-narkar — COMMENTED** [review]
Thanks for working on this @LionOnTheChase. Few comments inline.

**Comment by ashutosh-narkar** [comment]
Closing this for now. Feel free to pick this back up and address the comments when you get a chance. Thanks for looking into this.

**Comment by LionOnTheChase** [comment]
@ashutosh-narkar  , thank you for your patience. I have addressed your comments and created a new PR https://github.com/open-policy-agent/opa-envoy-plugin/pull/462 . 

Kindly review

**Pull request closed without merging**